src/watchmaker/workers/salt.py

```diff
@@ -255,6 +256,21 @@ def _build_salt_formula(self, extract_dir):
                 to_directory=extract_dir
             )
 
+        bundled_content = os.sep.join(
+            (static.__path__[0], 'salt', 'content')
+        )
+        for subdir in next(os.walk(bundled_content))[1]:
+            if not subdir.startswith('.') and \
```

Comment:
The spacing might be cleaner by wrapping the conditions in parens. What do you think? 

```suggestion
            if (
                not subdir.startswith('.') and
                not os.path.exists(os.sep.join((extract_dir, subdir)))
            ):
```